packages/stable/indent-runtime-aws-lambda/src/index.ts

```diff
@@ -30,3 +36,46 @@ export function getLambdaHandler({
 
   return handler
 }
+
+async function getSecret(secretName) {
+  switch (process.env.SECRETS_BACKEND) {
+    case "aws-secrets-manager": {
+      // for Secrets Manager, multiple secret values can be stored in one AWS Secret.
+      // When using Secrets Manager, store the AWS Secret name in the env var, and
+      // use the integration secret name as a key inside of that.
+
+      if (process.env[secretName] == undefined) {
+        // this secret is not in use for this configuration
+        // ex: OKTA_TOKEN when using JWK
+        return
+      }
+
+      // Call the secrets manager sidecar to get the secret
+      const res = await axios.get(
+        `http://localhost:2773/secretsmanager/get?secretId=${process.env[secretName]}`,
+        { headers: { 'X-Aws-Parameters-Secrets-Token': process.env.AWS_SESSION_TOKEN } }
+      )
+      const awsSecret = JSON.parse(res.data.SecretString)
+      console.log("got secret: ", secretName, awsSecret[secretName])
+      return awsSecret[secretName]
+    }
+  }
+}
+
+async function loadSecrets(integrations) {
```

Comment:
see above -- need a good way to prevent loadSecrets from being run multiple times (this implementation is of course a giant race condition waiting to happen).